tests/migration_test.go

```diff
@@ -1753,10 +1753,10 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 					By("Waiting for VMI to disappear")
 					tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 				},
-					table.Entry("[QUARANTINE][owner:@sig-storage][test_id:2226] with ContainerDisk", newVirtualMachineInstanceWithFedoraContainerDisk),
-					table.Entry("[QUARANTINE][owner:@sig-storage][sig-compute][test_id:2731] with OCS Disk (using ISCSI IPv4 address)", newVirtualMachineInstanceWithFedoraOCSDisk),
+					table.Entry("[test_id:2226] with ContainerDisk", newVirtualMachineInstanceWithFedoraContainerDisk),
+					table.Entry("[test_id:2731] with OCS Disk (using ISCSI IPv4 address)", newVirtualMachineInstanceWithFedoraOCSDisk),
 				)
-				It("[QUARANTINE][sig-compute][test_id:3241]should be able to cancel a migration right after posting it", func() {
+				It("[test_id:3241]should be able to cancel a migration right after posting it", func() {
```

Comment:
ah, i forgot about our new quarantine policy https://github.com/kubevirt/kubevirt/blob/master/docs/quarantine.md#getting-tests-out-of-quarantine

apparently we'd need to introduce this fix, then monitor the quarantined tests for two weeks before taking away the QUARANTINE marker.

Reply:
ok, I've removed it and will post it as a separate PR after this gets in..